src/clr-addons/treetable/treetable-row.scss

```diff
@@ -0,0 +1,119 @@
+clr-dg-row.clr-row-clickable {
+  cursor: pointer;
+}
+
+clr-treetable .datagrid clr-dg-cell {
+  flex-basis: 10%;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+clr-treetable .datagrid .datagrid-body clr-tt-row .clr-tt-node-content.datagrid-row {
+  border-top: 1px solid #ddd;
+}
+
+/* Indentation for first cells in rows */
```

Comment:
write nested CSS for this to make this way shorter

Reply:
good point, thanks for the tip